javascript/javascript3/week1/weather.js

```diff
@@ -0,0 +1,23 @@
+let button = document.querySelector(".button");
+let inputValue = document.querySelector(".inputValue");
+let nameValue = document.querySelector(".name");
+let desc = document.querySelector(".desc");
+let temp = document.querySelector(".temp");
+button.addEventListener("click", function () {
```

Comment:
The code does not tell much about what its doing. In programming we create abstractions such as functions or classes that deal with separate things and are named such that they tell us what they do.

Reply:
See if you can create atleast 2-3 functions that each deal with a separate responsibility to make your code more transparant. 
Also think about the names. 
Names like inputValue, button, nameValue, desc, temp are completely uninformative. You might as well call them a,b,c,d,e